analysis/src/main/java/org/hps/analysis/dataquality/MonsterEventAnalysis.java

```diff
@@ -183,4 +179,4 @@ public void process(EventHeader event){
             }
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
I really like all this clean up, but don't understand why these new lines with closing braces were added. If you can explain this I am ready to approve.